New content for support switch over

pocketstefan opened the pull request:
New email addresses and ways to get help now support is with CCS 

Could you send me a preview of the page please? Thank you! 

https://trello.com/c/EgWNENMj/101-revamp-help-page-content

lfdebrux commented:
![image](https://user-images.githubusercontent.com/503614/60582705-3b22f700-9d81-11e9-87d5-807e77d84e2c.png)

katstevens commented:
We'll need `<ul class="list-bullet">` or similar, I think

pocketstefan commented:
error error.

I have gone in an fixed the file so should be formatting correctly know.
Have also added text under the MI heading and fixed the link text in the
final section.

Stefan

On Wed, 3 Jul 2019 at 10:58, Kat <notifications@github.com> wrote:

> We'll need <ul class="list-bullet"> or similar, I think
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/906?email_source=notifications&email_token=AGWA2K55BYLP4JOFNBO3G3LP5RZ5JA5CNFSM4H44WXMKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZD6GJA#issuecomment-508027684>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGWA2KYID6XEKRYDW73AR3LP5RZ5JANCNFSM4H44WXMA>
> .
>


-- 
Stefan Nicolaou | Content Designer, Digital Marketplace
Government Digital Service | Cabinet Office
07825 865240
stefan.nicolaou@digital.cabinet-office.gov.uk
The White Chapel Building • 10 Whitechapel High Street • London • E1 8QS

lfdebrux commented:
I've updated my earlier comment

pocketstefan commented:
Thanks Laurence. I've had to tinker another link. Does the spacing between
headings seem off to you? Not sure how to fix.

Stefan

On Wed, 3 Jul 2019 at 11:53, Laurence de Bruxelles <notifications@github.com>
wrote:

> I've updated my earlier comment
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/906?email_source=notifications&email_token=AGWA2K7VTPZNJSTHPLCAKJLP5SAK5A5CNFSM4H44WXMKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZECIKQ#issuecomment-508044330>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGWA2K5UQ33BDAOW52IODZLP5SAK5ANCNFSM4H44WXMA>
> .
>


-- 
Stefan Nicolaou | Content Designer, Digital Marketplace
Government Digital Service | Cabinet Office
07825 865240
stefan.nicolaou@digital.cabinet-office.gov.uk
The White Chapel Building • 10 Whitechapel High Street • London • E1 8QS

lfdebrux commented on app/templates/help/index.html:
```diff
@@ -20,35 +20,42 @@
+  <li>selling your services, for example lowering your prices</li>
+  <li>buying services, for example evaluating suppliers</li>
+  <li>updating your account </li>
+</ul>
```
```suggestion
</ul>
</p>
```

lfdebrux commented on app/templates/help/index.html:
```diff
@@ -20,35 +20,42 @@
+         
+        <p><a href="https://www.gov.uk/guidance/buying-and-selling-on-the-digital-marketplace">Read the guidance</a> for information about:</p>
+      
+       <ul class="list-bullet"> 
```
```suggestion
       <p>
       <ul class="list-bullet"> 
```

lfdebrux commented on app/templates/help/index.html:
```diff
@@ -20,35 +20,42 @@
+        <li>the G-Cloud and Digital Outcomes and Specialists frameworks</li>
+        <li>how to apply to join as a supplier</li>
+        <li>how to buy</li> 
+      </ul> 
```
```suggestion
      </ul>
      </p>
```
